### DISCUSSION TITLE
Can you override a dep thats ignored in `ignoreDeps` in a specific `packageRule`?

### DISCUSSION THREAD

**Discussion by samjcombs** [renovatebot/renovate#20050] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

a way to override an ignored dep in one packageRule? 

### What you tried so far.

-

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
You cannot. So don't use ignoreDeps and instead disable them using packageRules so that they can be reenabled using packageRules

**Reply by samjcombs** [reply]
thanks for the reply,  I have a bunch of rules, so my use case would be to use `ignoreDeps` so it ignores them in all of them that run monthly, but every 3 months I want it to check for a small list of ones that I know I cant update until some downstream dep is updated. 

Specifically I can't update `node-fetch` to ^3 until its fully compatible with jest being used with typescript.  My goal is to ignore node-fetch majors until that known issue gets resolved. 

Similarly there is a known bug in Jest 29 so I want to silence jest majors until that bug gets resolved.  

I run it monthly across a bunch of repos, so my idea was to create a batch of these specific ones that will run quarterly so we dont lose track.  

Is there any config/feature that renovate has that can say, ignore this dep until a certain time, or ignore it until some other trigger? 

Anyways, just mind dumping here, no problem if theres no solution, i can finesse the configs to do whats desired. 

One last question, it would be great to annotate a custom comment beside a dep in the PR.  Like if `node-fetch` appears I could comment and say 'cant upgrade until [some issue](#) is resovled' thus pointing the user to the issue to check on it. 

Thanks!

**Reply by rarkins** [reply]
I would recommend you use dependencyDashboardApproval for these so that they're suppressed but not hidden

**Reply by samjcombs** [reply]
thanks for that tip, thats a bit of a heavy lift to set that up as we have a central config that gets delievered to a dozen repos.  My idea is to go into a specific repo and extend the prBodyColumns with a custom Notes field that has a conditional to see if a certain major dep is found and then print the note there for the next person reviewing the PR. 

But this didn't work, any ideas what I'm doing wrong?

```json
{
  "extends": ["local>shared/renovate-config//renovate-presets/gql.json5"],
  "prBodyDefinitions": {
    "Notes": "{{#if and isMajor (containsString depName 'node-fetch')}}Attempted upgrade on 1/26/23: Upgrading to ^3 oddly causes issues with Jest when using Typescript.  Upgrade with caution... Track issues here: https://github.com/node-fetch/node-fetch/issues/1289{{/if}}{{#if and isMajor (containsString depName 'hot-shots')}}Attempted upgrade on 1/26/23: Can't upgrade due to peerDependencies still requiring ^8.{{/if}}"
  },
  "prBodyColumns": [
    "Package",
    "Type",
    "Update",
    "Change",
    "Notes"
  ]
}
```

**Reply by samjcombs** [reply]
can i put two ifs beside eachother?  and two args to and's like that?

**Reply by viceice** [reply]
you maybe need to add additional parenthesis around the `and` expression
